documentation/Set-PnPTenant.md

```diff
@@ -1367,7 +1367,7 @@ Accept wildcard characters: False
 ```
 
 ### -EnableModernListTemplateIds
-Guids of out of the box modern list templates to show when creating a new list
+Guid'ss of out of the box modern list templates to show when creating a new list
```

Comment:
Same here, the previous version is the correct one.

Reply:
updatee